cxx-sensors/src/test/java/org/sonar/cxx/sensors/tests/xunit/CxxXunitSensorTest.java

```diff
@@ -86,14 +88,31 @@ public void shouldReadXunitReport() {
         tuple(CoreMetrics.TEST_EXECUTION_TIME_KEY, 0L));
   }
 
+  @Test
+  public void shouldIgnoreSubModuleXunitReport() {
```

Comment:
Please split into 2 tests.
- without parent
- with parent

Reply:
Thank you for the comments; the test "without parent" already exists (see `shouldReadXunitReport()`). I'll remove the unrelated (and duplicated) asserts.